docs/pages/configuration/profiles/patches.mdx

```diff
@@ -115,13 +118,19 @@ profiles:
   - op: replace
     path: images.backend.image
     value: john/stagingbackend
+  - op: replace
+    path: deployments.name=backend.helm.values.containers[0].image
+    value: john/stagingbackend
   - op: remove
     path: deployments.name=backend.helm.values.containers[1]
 - name: production
   patches:
   - op: replace
     path: images.backend.image
     value: john/prodbackend
+  - op: replace
+    path: deployments.name=backend.helm.values.containers[0].image
```

Comment:
Should not use index 0  since we just recommend using selectors just above it. Same for the other one one in this example.

Reply:
I've update the examples after the selectors recommendation to use `deployments.name=backend.helm.values.containers.image=john/devbackend.image` or similar instead of indices.

Please have another look.